test/e2e/kubelet.go

```diff
@@ -98,7 +98,7 @@ func waitTillNPodsRunningOnNodes(c clientset.Interface, nodeNames sets.String, p
 // In case a given label already exists, it overwrites it. If label to remove doesn't exist
 // it silently ignores it.
 // TODO: migrate to use framework.AddOrUpdateLabelOnNode/framework.RemoveLabelOffNode
-func updateNodeLabels(c clientset.Interface, nodeNames sets.String, toAdd, toRemove map[string]string) {
+func UpdateNodeLabels(c clientset.Interface, nodeNames sets.String, toAdd, toRemove map[string]string) {
```

Comment:
Can you please move this function to framework? I don't want to depend on test/e2e .